### PR TITLE
Add tip on setting tokenizer attributes

### PR DESCRIPTION
**Pull request by Rocketknight1**
This PR adds a quick tip to the chat template docs on setting tokenizer attributes (after some discussion on Slack)

### CODE REVIEW

File: docs/source/en/chat_templating.md [hunk 1]
```diff
@@ -343,6 +343,15 @@ tokenizer.push_to_hub("model_name")  # Upload your new template to the Hub!
 The method [`~PreTrainedTokenizer.apply_chat_template`] which uses your chat template is called by the [`ConversationalPipeline`] class, so 
 once you set the correct chat template, your model will automatically become compatible with [`ConversationalPipeline`].
 
+<Tip>
+If you're fine-tuning a model for chat, in addition to setting a chat template, you should probably add any new chat
```
Comment (amyeroberts) [inline docs/source/en/chat_templating.md:347]: I'd remove `probably` here - it's sounds like the tip is unsure! 

```suggestion
If you're fine-tuning a model for chat, in addition to setting a chat template, you should add any new chat
```

**Review by amyeroberts — APPROVED** [review]
Thanks for adding!

File: docs/source/en/chat_templating.md (hunk 1 above)
Comment (ArthurZucker) [reply docs/source/en/chat_templating.md:347]: +1 here! But some models are trained without these tokens as special 😞

**Review by ArthurZucker — APPROVED** [review]

File: docs/source/en/chat_templating.md (hunk 1 above)
Comment (amyeroberts) [reply docs/source/en/chat_templating.md:347]: Ah, OK. So maybe something like: `"If your fine-tuning a model for chat and it was trained with special tokens, you should..."`

Comment (Rocketknight1) [reply docs/source/en/chat_templating.md]: I think it's okay to leave it as-is - this is advice for people who are about to fine-tune a model, in which case there's still time for them to add the special tokens!

I agree it's too late once the model is already trained, of course

**Pull request merged**